django-backend/fecfiler/settings/base.py

```diff
@@ -28,6 +28,9 @@
 APPEND_SLASH = False
 
 LOG_FORMAT = env.get_credential("LOG_FORMAT", LINE)
+ENABLE_PL_SQL_LOGGING = get_boolean_from_string(
```

Comment:
Is it overkill to add a comment here saying something like "Never set this to true for dev/stage/test/prod"?

Reply:
Perfect, thank you!